src/Microsoft.Diagnostics.Monitoring.EventPipe/Configuration/AggregateSourceConfiguration.cs

```diff
@@ -25,14 +36,37 @@ public override IList<EventPipeProvider> GetProviders()
 
         public override bool RequestRundown
         {
-            get => _configurations.Any(c => c.RequestRundown);
+            get => ComputeRundownKeyword() != 0;
             set => throw new NotSupportedException();
         }
 
         public override long? RundownKeyword
         {
-            get => _configurations.Select(c => c.RundownKeyword).Aggregate((x, y) => x | y);
+            get
+            {
+                long keywords = ComputeRundownKeyword();
+                return ((keywords == 0) || (keywords == defaultRundownKeyword)) ? null : keywords;
```

Comment:
How about we set RundownKeyword to an explicit value anytime RequestRundown is true, then in DiagnosticClient if we observe that rundown keyword == default we can send the older event rather than the new one?